core/src/main/java/com/hcl/ecomm/core/services/impl/ProductServiceImpl.java

```diff
@@ -34,8 +34,8 @@
 
 	String scheme = "http";
 	String responseStream = null;
-	JsonArray productJsonArray = null;
```

Comment:
@Subrat-Dash-416 16 Why this class changed?

Reply:
same as above